internal/testrunner/runners/system/servicedeployer/custom_agent.go

```diff
@@ -154,5 +154,5 @@ func (d *CustomAgentDeployer) loadComposeDefinitions() ([]string, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "can't locate Docker Compose file for Custom Agent deployer")
 	}
-	return []string{locationManager.DockerCustomAgentDeployerYml(), d.cfg}, nil
+	return []string{d.cfg,locationManager.DockerCustomAgentDeployerYml()}, nil
```

Comment:
Could you please add a test package that uses a dockerfile into `test/packages/with-custom-agent`?